webapp/src/main/java/ar/edu/itba/paw/webapp/exceptions/ApplicationExceptionMapper.java

```diff
@@ -0,0 +1,21 @@
+//package ar.edu.itba.paw.webapp.exceptions;
+//
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+//
+//import javax.ws.rs.core.MediaType;
+//import javax.ws.rs.core.Response;
+//import javax.ws.rs.ext.ExceptionMapper;
+//import javax.ws.rs.ext.Provider;
+//
+//@Provider
+//public class ApplicationExceptionMapper implements ExceptionMapper<Exception>{
+//
+//  private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationExceptionMapper.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)